about.html

```diff
@@ -0,0 +1,127 @@
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </nav>
+      <div class="dropdown">
+        <ul id="options" class="dropdown-content">
+          <li><a href="./index.html">Home</a></li>
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </div>
+      <div>
+        <p>CC Campaing</p>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="about_headline">
+      <div class="container">
+        <div>
+          <h1>Talent Land Mexico 2023</h1>
+        </div>
+        <div>
+          <p>Talent Land 2023 will be dedicated to the development of talent, so that the participants can meet with
+            experts from different areas who will share their experience and knowledge.
+            This event is a place for creative minds, entrepreneurs and young talents to find new ideas, new people and
+            new technologies to improve their personal and professional lives.
+            Talent Land 2023 is underway and it will be unique.</p>
+        </div>
+        <div>
+          <p>Please contact us per email for any further questions about CC Talent Land Mexico.</p>
+          <p>info@talent-network.org</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="about_talent container">
+      <h2>Jalisco Talent Land </h2>
+      <hr>
+      <p>Since the first Talent Land we managed to bring together creativity and technology, for the next 2023, the
+        machinery has been set in motion to create new emotions, promote the development of talent and promote
+        entrepreneurship among young people.
+        With the same entrepreneurial spirit that has allowed us to carry out unforgettable experiences on a global
+        level, in April 2023 we will discover new ways to change the world with talent.
+        “Talent Land 2023: the future is yours”
+        Get ready!</p>
+      <div>
+        <img src="./assets/logo-talent-land.png" alt="">
+      </div>
+    </section>
+
+    <section class="past-editions">
+      <h2>See the past editions of Talent Land</h2>
+      <p>Take a look at the last two Talent Land Mexico in Expo Guadalajara</p>
+      <div class="past_images">
+        <div class="box"><img class="image" src="./images/TalentLand_2021.jpg" alt="">
+          <div class="middle">
+            <div class="text">Talent Land 2021</div>
+          </div>
+        </div>
+        <div class="box"><img class="image" src="./images/TalentLand_2022.jpg" alt="">
+          <div class="middle">
+            <div class="text">Talent Land 2022</div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+  </main>
+  <footer class="container-fluid">
+    <div>
+      <p>© Copyright 2023</p>
+    </div>
+    <div>
+      <p>Talent Network.</p>
+      <p>Todos los derechos reservados. </p>
+    </div>
+  </footer>
```

Comment:
- [ ] The background color for footer on the about page is dark while your design carries different background color. Kindly fix that too.

### Your Design
![your-footer](https://user-images.githubusercontent.com/26597432/233480595-9061353b-ba30-4b73-901a-34bd21bf551f.png)

### Guiding Template
![temp-pix-footer](https://user-images.githubusercontent.com/26597432/233480620-a1819de3-3512-4625-a8c4-014883f37d39.png)

- [ ] The partner section is missing on your about page. Kindly add that to your page

### Your Design
![your-about-partner-missing](https://user-images.githubusercontent.com/26597432/233481654-e708e8d8-70e7-4da1-a808-1128d75d1788.png)

### Guiding Template
![temp-partner-missing](https://user-images.githubusercontent.com/26597432/233481676-e0b3cbb7-a869-460a-b65f-03fe574dc465.png)